eth/backend.go

```diff
@@ -351,6 +351,10 @@ func makeExtraData(extra []byte) []byte {
 	return extra
 }
 
+func (s *Ethereum) PeerCount() int {
```

Comment:
comment to document the function

Reply:
[Done](https://github.com/maticnetwork/bor/pull/945/commits/abe87af0bb99b7995b621af5eb41d50b4a964e0f).